src/invalid.ts

```diff
@@ -0,0 +1,79 @@
+import {SignalRef} from 'vega';
+import {ScaleChannel} from './channel';
+import {Mark, MarkDef} from './mark';
+import {isObject} from 'vega-util';
+
+/**
+ * Mixins for Vega-Lite Spec's Mark Definiton (to add mark.invalid)
+ */
+export interface MarkInvalidMixins {
+  /**
+   * Invalid data mode for marks, which defines how the visualization should represent invalid values (`null` and `NaN` in continuous scales without defined output for invalid values) in the marks and their scale domains.
+   *
+   * -  `"filter"` — *Exclude* all invalid values from the visualization's *marks* and *scales*.
+   * For path marks (for line, area, trail), this option will create paths that connect valid points, as if the points with invalid values do not exist.
+   *
+   * - `"break-paths"` —
+   * Break path marks (for line, area, trail) at invalid values.  For non-path marks, this is equivalent to `"filter"`.
```

Comment:
This comment doesn't say anything about the "ignore in domain" thing. Ppl have to look at `"break-paths-keep-domains"` to guess that this is actually `"break-paths-ignore-domain"`. I'd prefer being explicit -- name this as "break-paths-ignore-domain" and add corresponding comment about the domain behavior.